packages/taucmdr/cli/commands/trial/tests/test_export.py

```diff
@@ -54,6 +54,7 @@ def test_export_tau_profile(self):
         export_file = expr['name'] + '.trial0.ppk'
         self.assertTrue(os.path.exists(export_file))
 
+    @tests.skipIf(util.which('nvc'), "NVHPC compilers do not work with sqlite")
```

Comment:
@nchaimov and @vikram8128 what's the issue with sqlite and NVHPC compilers?

Reply:
The TAU sqlite plugin does not link when built with NVHPC due to a missing symbol that makes reference to C++17 even when `-std=c++11` is passed. This appears to be a compiler bug in NVHPC.

Reply:
Might be nice to have a comment to that effect